src/Facade/ResourceBuilders/MechPartSourceResourceBuilder.cs

```diff
@@ -101,8 +106,47 @@ public MechPartSourceResource Build(MechPartSource model)
                             LibraryRef = model.LibraryRef,
                             FootprintRef = model.FootprintRef,
                             RkmCode = model.RkmCode,
-                            CapictanceLetterAndNumeralCode = model.CapacitanceLetterAndNumeralCode,
-                            Links = this.BuildLinks(model).ToArray()
+                            CapacitanceLetterAndNumeralCode = model.CapacitanceLetterAndNumeralCode,
+                            PartCreatedBy = model.PartCreatedBy?.Id,
+                            PartCreatedByName = model.PartCreatedBy?.FullName,
+                            PartCreatedDate = model.PartCreatedDate?.ToString("o"),
+                            VerifiedBy = model.VerifiedById,
+                            VerifiedByName = model.VerifiedBy?.FullName,
+                            VerifiedDate = model.VerifiedDate?.ToString("o"),
+                            QualityVerifiedBy = model.QualityVerifiedById,
+                            QualityVerifiedByName = model.QualityVerifiedBy?.FullName,
+                            QualityVerifiedDate = model.QualityVerifiedDate?.ToString("o"),
+                            McitVerifiedBy = model.McitVerifiedById,
+                            McitVerifiedByName = model.McitVerifiedBy?.FullName,
+                            McitVerifiedDate = model.McitVerifiedDate?.ToString("o"),
+                            ApplyTCodeBy = model.ApplyTCodeId,
+                            ApplyTCodeByName = model.ApplyTCodeBy?.FullName,
+                            ApplyTCodeDate = model.ApplyTCodeDate?.ToString("o"),
+                            RemoveTCodeBy = model.RemoveTCodeId,
+                            RemoveTCodeByName = model.RemoveTCodeBy?.FullName,
+                            RemoveTCodeDate = model.RemoveTCodeDate?.ToString("o"),
+                            CancelledBy = model.CancelledById,
+                            CancelledByName = model.CancelledBy?.FullName,
+                            CancelledDate = model.DateCancelled?.ToString("o"),
+                            PurchasingQuotes = model.PurchasingQuotes?.Select(
+                                (q, i) =>
+                                    {
+                                        var resource = this.purchasingQuotesResourceBuilder.Build(q);
+                                        resource.Id = i;
```

Comment:
Do these ids need to be the iterator?

Reply:
Yeah, I think so. The underlying entity Id is supplierId or something, but since you can change these in the table it doesn't make for a very reliable Id for react map keys.

Reply:
Is this better done in the client if the fields are only used by React? It seems weird to put this in the resource when its not really a field on the object (& just used in a react array)